components/eventTypeAvailTimes.jsx

```diff
@@ -39,72 +56,132 @@ export default () => {
   }, []);
 
   return (
-    <div className="event-container">
-      <h5>Check Availability for "{eventType.name}"</h5>
-      <h6 className="event-type-avail-banner">
+    <div className="event-avail-selection-box">
+      <h5>"{eventType.name}"</h5>
+      <h6 className="event-avail-header">
         Click below to see availability for this event type by start date
       </h6>
       <div>
         <strong>
-          *Note: Time range will be 7 days from your chosen start date
+          **Selected date/time must be in future. (Time range will be 7 days from
+          your chosen start date.)**
         </strong>
       </div>
-      <div className="event-avail-calendar">
-        <DatePicker
-          selected={date}
-          placeholderText="CLICK HERE (selected date and time must be in the future)"
-          minDate={new Date()}
-          dateFormat="MMMM d, yyyy"
-          onChange={(date) => {
-            setDate(date);
-            setShowTIme(true);
-            let copyDate = new Date(date.getTime());
-            let endDate = new Date(copyDate.setDate(copyDate.getDate() + 7));
-            setFinalDateMillisec(new Date(date).getTime());
-            setQueryParams(
-              `?start_time=${date.toISOString()}&end_time=${endDate.toISOString()}&event_type=${eventUri}`
-            );
-          }}
-        />
-      </div>
-      {showTime && (
-        <div>
-          <input
-            type="time"
-            id="selected-time"
-            name="selected-time"
-            defaultValue="09:00"
-            step="900"
-            onChange={(event) => {
-              let dateToModify = date.toString().split(' ');
-              let time = event.target.value;
-              dateToModify[4] = time;
-              let dateWithTime = new Date(dateToModify.join(' '));
-              setDate(dateWithTime);
-              let copyDate = new Date(dateWithTime.getTime());
-              let endDate = new Date(copyDate.setDate(copyDate.getDate() + 7));
-              setShowSubmit(true);
-              setFinalDateMillisec(new Date(dateWithTime).getTime());
+      <div className="date-and-time-pickers">
+        <div className="date-picker">
+          <DatePicker
+            selected={date}
+            placeholderText="        CHOOSE DATE        "
```

Comment:
It should be possible to adjust the spacing via css if you wanted to

Reply:
Actually, I don't need the odd spacing. I did that before I added the "input" selector in the CSS file. The centering via the input selector takes care of centering that text!